src/CMakeLists.txt

```diff
@@ -2,7 +2,22 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-cmake_minimum_required(VERSION 3.10)
+option(
+  BUILD_BASIS_COMPRESSOR
+  "Wether or not to build the basis compressor.  Loading basis compressed meshes does NOT require this."
+  OFF
+)
+
+if(BUILD_BASIS_COMPRESSOR)
+  if(CMAKE_VERSION VERSION_LESS "3.13")
+    message("Building the basis compressor requires a newer version of cmake.")
+    message("You can get one with pip install cmake")
+  endif()
+  cmake_minimum_required(VERSION 3.13)
```

Comment:
This command could be directly in `utils/imageconverter/CMakeLists.txt`, I think, that would make it more modular -- the minimum CMake version could change based on what all subdirs are added, while the top-level file requires still just 3.10.